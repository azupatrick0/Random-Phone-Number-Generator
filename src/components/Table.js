import React from 'react';

const Table = ({ serialNumber, tableValue}) => (
  <div style={{
    borderRadius: 4,
    boxShadow: '1px 1px 4px plum',
    display: 'flex',
    textAlign: 'center',
    width: '40vw',
    borderBottom: '1px solid white'
  }}>
    <div style={{ fontSize: 30, fontWeight: 'bold', width: '30vw', backgroundColor: 'gray', color: 'white' }}>{serialNumber}</div>
    <div style={{ fontSize: 30, fontWeight: 'bold', width: '60vw',  backgroundColor: 'plum', color: 'white' }}>{tableValue}</div>
  </div>
)

export default Table;
