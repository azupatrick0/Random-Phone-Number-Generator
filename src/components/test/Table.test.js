import React from 'react';
import Table from '../Table';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';

configure({ adapter: new Adapter() });

it('renders Table component without crashing', () => {
  const wrapper = shallow(<Table />);
  expect(wrapper).toMatchSnapshot();
});
