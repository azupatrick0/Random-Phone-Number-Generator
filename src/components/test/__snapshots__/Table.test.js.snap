// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Table component without crashing 1`] = `ShallowWrapper {}`;
